spec/spec_helper.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.configure do |config|
```

Comment:
Do you know what this configuration did?

Reply:
Doc says that rspec-expectations config goes here. I guess we may keep  it, at least in case of future updates of rspec functionality

Reply:
So you don't understand what it does

Reply:
If we don't understand something, it's not trigger to remove something.

Reply:
Removed `spec_helper.rb`